src/components/application_manager/test/mobile_message_handler_test.cc

```diff
@@ -197,8 +197,8 @@ class MobileMessageHandlerTest : public testing::Test {
     MobileMessage message_to_send = CreateMessageForSending(
         protocol_version, function_id, correlation_id, connection_key, data);
     // Act
-    RawMessage* result_message =
-        MobileMessageHandler::HandleOutgoingMessageProtocol(message_to_send);
+    std::shared_ptr<RawMessage> result_message(
```

Comment:
@SNiukalov Can't see any sharing here. What is the reason to create shared_ptr?

Reply:
@AByzhynar 
Fixed in:https://github.com/LuxoftSDL/sdl_core/pull/41/commits/be4a738c2392d04c4fd0cd9a2dee39ce76b94c6e